src/nanosvgrast.h

```diff
@@ -25,6 +25,8 @@
 #ifndef NANOSVGRAST_H
 #define NANOSVGRAST_H
 
+#include "nanosvg.h"
```

Comment:
With this change, it's no longer possible to include `nanosvgrast.h` after including `nanosvg.h` as it leads to duplicate symbols:
```
$ cat << EOF > nanosvg.cc
#include "stdio.h"
#include "string.h"
#include "math.h"
#define NANOSVG_ALL_COLOR_KEYWORDS
#define NANOSVG_IMPLEMENTATION
#include "nanosvg.h"
#define NANOSVGRAST_IMPLEMENTATION
#include "nanosvgrast.h"
EOF

$ g++ nanosvg.cc
In file included from nanosvgrast.h:28,
                 from nanosvg.cc:8:
nanosvg.h:219:12: error: redefinition of ‘int nsvg__isspace(char)’
  219 | static int nsvg__isspace(char c)
      |            ^~~~~~~~~~~~~
In file included from nanosvg.cc:6:
nanosvg.h:219:12: note: ‘int nsvg__isspace(char)’ previously defined here
  219 | static int nsvg__isspace(char c)
      |            ^~~~~~~~~~~~~
nanosvg.h:224:12: error: redefinition of ‘int nsvg__isdigit(char)’
  224 | static int nsvg__isdigit(char c)
      |            ^~~~~~~~~~~~~
nanosvg.h:224:12: note: ‘int nsvg__isdigit(char)’ previously defined here
  224 | static int nsvg__isdigit(char c)
      |            ^~~~~~~~~~~~~
nanosvg.h:229:26: error: redefinition of ‘float nsvg__minf(float, float)’
  229 | static NSVG_INLINE float nsvg__minf(float a, float b) { return a < b ? a : b; }
      |                          ^~~~~~~~~~
nanosvg.h:229:26: note: ‘float nsvg__minf(float, float)’ previously defined here
  229 | static NSVG_INLINE float nsvg__minf(float a, float b) { return a < b ? a : b; }
      |                          ^~~~~~~~~~
nanosvg.h:230:26: error: redefinition of ‘float nsvg__maxf(float, float)’
  230 | static NSVG_INLINE float nsvg__maxf(float a, float b) { return a > b ? a : b; }
      |                          ^~~~~~~~~~
nanosvg.h:230:26: note: ‘float nsvg__maxf(float, float)’ previously defined here
  230 | static NSVG_INLINE float nsvg__maxf(float a, float b) { return a > b ? a : b; }
      |                          ^~~~~~~~~~
...
```
and dozen more errors.
```

Reply:
That include should not be there, my bad reviewing it. @tamasmeszaros would you mind making another PR fixing that?

Reply:
Yes, but I would not remove the include but rather extend the include guards for the whole files for nanosvg.h and nanosvgrast.h.